README.md

```diff
@@ -1,2 +1,178 @@
+
+This repository contains the Frontend part of the Clock of Change in form of a Wordpress plugin.
+
+The purpose of the Clock of Change Frontend is to display the visual part of the Clock of Change, 
+handle user interactions (especially registrations) and provide a basic administration surface for the Clock of Change through Wordpress.
+
+The Clock of Change Frontend depends on the [Clock of Change API](https://github.com/Human-Connection/Clock-of-Change-API),
+which acts as the Backend and takes care of storing and processing the data.
+
+To find out more about the Clock of Change and Human Connection - the network behind it - visit https://human-connection.org/en/.
+
+## Tech Stack
+
+* PHP: The Frontend uses mainly [PHP](http://php.net/) as programming language
+* Wordpress: The Frontend is running on [Wordpress](https://wordpress.org/download/) - the most used content management system
+* MySQL: Wordpress requires a [MySQL](https://www.mysql.com) database 
+* JavaScript / jQuery: We use [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript) and [jQuery](https://jquery.com/) for the UX, user interactions and animations
+* CSS: We use [CSS](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics) to style the Frontend
+* Composer: We use [Composer](https://getcomposer.org/) to manage (PHP) dependencies
+
+## Project Structure & Components
+
+**PROJECT STRUCTURE / DIRECTORIES**
+
+* `coc/`: Main directory for the Clock of Change Frontend Wordpress plugin
+* `coc/assets/`: Assets for the Frontend with subdirectories for css, images and js files
+* `coc/classes/`: Contains the PHP classes for the Frontend with the main class ClockOfChange.php 
+and core and shortcode classes in the respective subdirectories
+* `coc/config`: Contains the Wordpress options for the plugin
+* `coc/helper`: Contains helper classes
+* `coc/vendor/`: Contains the dependencies (packages) that are managed by composer
+* `coc/coc.php`: Main Wordpress plugin file
+
+**WORDPRESS**
+
+The Clock of Change Frontend is running as a Wordpress plugin. This is why we need a Wordpress system, 
+in which we can use the Clock of Change Frontend as a plugin.
+
+When using the Docker installation, we provide a full Wordpress system with the Docker container.
+Otherwise you need to setup a Wordpress system manually. To setup Wordpress you need to have a webserver with PHP and a MySQL database. 
+Then just follow the famous [Wordpress 5-minute installation](https://codex.wordpress.org/Installing_WordPress#Famous_5-Minute_Installation).
+
+
+## Installation
+
+**PREREQUESITES**
+
+Before starting the installation you need to make sure you have a recent version of [Git](https://git-scm.com/), [Composer](https://getcomposer.org/) installed. 
+E.g. we have the following versions:
+```
+$ git --version
+git version 2.14.2.windows.1
+$ composer --version
+Composer version 1.8.4 2019-02-11 10:52:10
+
+Git: 2.14.2
+Composer: 1.8.4
```

Comment:
1. `composer` is not a prerequisite for the docker installation. If you put this section on the same level as `Docker Installation` I would believe I have to install `compose` first.

2. I don't think we need to specify `git` as a requirement. You're on Github, so people should have `git`.
3. Please move the entire section **PREREQUESITES** under => **LOCAL INSTALLATION & USAGE**